ludwig/combiners/combiners.py

```diff
@@ -60,8 +60,8 @@ class ConcatCombinerConfig:
     num_fc_layers: int = schema.NonNegativeInteger(default=0)
     fc_size: int = schema.PositiveInteger(default=256)
     use_bias: bool = True
-    weights_initializer: str = schema.InitializerOptions(default='glorot_uniform')
-    bias_initializer: str = schema.InitializerOptions(default='zeros')
+    weights_initializer: Union[str, Dict] = schema.InitializerOrDict(default='glorot_uniform')
+    bias_initializer: Union[str, Dict] = schema.InitializerOrDict(default='zeros')
```

Comment:
Out of curiosity, what's an example of a bias initializer configured using a `dict`?

Reply:
Don't have a concrete example for you 😅 but it's explicity supported by the [developer guide](https://ludwig-ai.github.io/ludwig-docs/user_guide/configuration/combiners/#concat-combiner), traces to [TF documentation for initializer parameters](https://www.tensorflow.org/api_docs/python/tf/keras/initializers)